docs/deploy.md

```diff
@@ -0,0 +1,152 @@
+# Deploying M3DB
+
+## Deploying a single node
+
+Deploying a single-node cluster is a great way to experiment with M3DB and get a feel for what it
+has to offer. Our Docker image by default configures a single M3DB instance with an embedded etcd
```

Comment:
super nit: seems weird to mention etcd here but not below, and to mention the index below but not here. Maybe mention both in one place or both places?